.github/workflows/cypress.yaml

```
@@ -0,0 +1,96 @@
+name: Running Cypress Tests
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)